instrumentation/spring/spring-webflux/spring-webflux-5.3/library/src/main/java/io/opentelemetry/instrumentation/spring/webflux/v5_3/internal/WebClientHttpAttributesGetter.java

```diff
@@ -59,4 +59,16 @@ public String getServerAddress(ClientRequest request) {
   public Integer getServerPort(ClientRequest request) {
     return request.url().getPort();
   }
+
+  @Nullable
+  @Override
+  public String getErrorType(
+      ClientRequest request, @Nullable ClientResponse response, @Nullable Throwable error) {
+    // if both response and error are null it means the request has been cancelled -- see the
+    // WebClientTracingFilter class
```

Comment:
Perhaps it should be documented in the instrumentation doc?

Reply:
Fair idea, will do. Thanks!